docs/src/modules/components/ComponentLinkHeader.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable react/jsx-no-target-blank */
+/* eslint-disable material-ui/no-hardcoded-labels */
```

Comment:
Most of the links are for content in English only. I haven't translated them to communicate this information upfront, so people know what to expect before clicking.

Reply:
Why is the rule not firing anymore?

>  I haven't translated them to communicate this information upfront

I don't understand this statement. What are you communicating upfront?

Reply:
> Why is the rule not firing anymore?

I get it. It only works for native looking labelling attributes. The other point still stands.

Reply:
> What are you communicating upfront?

That if the visitor doesn't speak English, visiting the link won't be helpful.